CapstoneProject/Models/MoodDetection.m

```diff
@@ -0,0 +1,82 @@
+//
+//  MoodDetection.m
+//  CapstoneProject
+//
+//  Created by Apoorva Chilukuri on 7/15/22.
+//
+
+#import "MoodDetection.h"
+
+@interface MoodDetection ()
+
+@property (strong, nonatomic) NSMutableArray *userTextWords;
+
+@end
+
+@implementation MoodDetection
+
+- (void) initializer{
+    
+    NSArray *sadKeyWords = @[@"sad", @"down", @"terrible", @"depressed"];
+    NSArray *boredKeyWords = @[@"bored", @"ok", @"tired", @"sleepy"];
+    NSArray *happyKeyWords = @[@"happy", @"great", @"esstatic", @"excited"];
+    NSArray *angryKeyWords = @[@"angry", @"mad", @"upset"];
+    
+    
+    self.stopWordsForMoods = [NSMutableDictionary dictionaryWithObjectsAndKeys:
+                                  sadKeyWords,    @"sad",
+                              boredKeyWords,  @"bored",
+                              happyKeyWords, @"happy",
+                              angryKeyWords, @"angry",
```

Comment:
We don't need to do this right now, but something we should consider using is to make an enum for different types.